packages/client-mcp/tests/test_resources_and_prompts.py

```
@@ -0,0 +1,431 @@
+def test_get_prompt_result_has_description() -> None:
+    result = prompts_mod.get_prompt('analyze-document', {'pipeline': 'p', 'query': 'q'})
+    assert result.description is not None
+    assert 'Analyze' in result.description
+
+
+# =============================================================================
+# Server handler registration (smoke tests)
+# =============================================================================
+
+
+async def test_server_registers_list_resources_handler(env_rocketride: None) -> None:
```

Ruff (0.15.7)

[warning] Unused function argument: `env_rocketride`

(ARG001)